geoips/interfaces/base.py

```diff
@@ -26,7 +27,21 @@
 from referencing import jsonschema as refjs
 from jsonschema.exceptions import ValidationError, SchemaError
 
-from geoips.errors import PluginError, PluginRegistryError
+from geoips.errors import (
+    EntryPointError,
+    PluginError,
+    PluginRegistryError,
+    PluginValidationError,
+)
+
+from geoips.geoips_utils import (
+    find_entry_point,
+    split_camel_case,
+    # get_all_entry_points,
```

Comment:
I'll remove those comments. I think I left them there in case they'd be used again but you're right in the point that I can remove them and they can be re-added later on if needed.

Reply:
Removed!